pre/pip-common.sh

```diff
@@ -27,9 +27,9 @@ _PIP_ANSIBLE_DEPS="$($PYTHON <<EOF
 import pip
 pip_major = int(pip.__version__.split('.')[0])
 if pip_major < 9:
-    print("pyyaml cryptography>=2.5,<3.4 jinja2<3 markupsafe<2 packaging<21")
+    print("pyyaml cryptography>=2.5,<3.4 jinja2<3 markupsafe packaging<21")
```

Comment:
In fact I'm wondering if we should revert that change

Reply:
yeah, looks like the latest version _is_ 2.1.0

Reply:
I think you messed up the changes ^^

we need to keep markupsafe<2 for pip < 9 but we still need you change markupsafe<2.1 for pip < 19